pulp_container/tests/unit/test_serializers.py

```diff
@@ -28,13 +28,13 @@ def setUp(self):
             f"{V3_API_ROOT}repositories/container/container-push/{self.push_repository.pk}/"
         )
         self.user = get_user_model().objects.create(username="user1", is_staff=False)
-        _set_current_user(self.user)
+        set_current_user(self.user)
```

Comment:
Did not know this was a thing. ;)
If we had properly imported that library, we'd need to drop the requirement now... XD